cli/tests/websocketstream_test.ts

```diff
@@ -0,0 +1,123 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+// deno-lint-ignore-file no-undef
```

Comment:
Now the linter doesn't run `no-undef` (due to #10652) so we can remove this directive.